### PR TITLE
calculate apr with less than 7 days data

### PR DESCRIPTION
**Pull request by 0xchewbacca**

### CODE REVIEW

**Comment by yellowgh0st** [comment]
@0xchewbacca Please fix build issue.

File: src/common/pricing/coingecko.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import axios from 'axios'
+
+const getPrice = async (coingeckoId) => {
+	const url = `https://api.coingecko.com/api/v3/simple/price?vs_currencies=usd&ids=${coingeckoId}`
```
Comment (0xchewbacca) [inline src/common/pricing/coingecko.js:4]: this is for kovan or just in case can't get data from uniswap subgraph

Comment (0xchewbacca) [inline src/common/pricing/coingecko.js:4]: this is for kovan or just in case can't get data from uniswap subgraph

Comment (yellowgh0st) [reply src/common/pricing/coingecko.js:4]: ok

**Review by yellowgh0st — APPROVED** [review]

**Pull request merged**